addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet_extended.js

```diff
@@ -1,5 +1,6 @@
 /** @odoo-module */
 
+/** @type {o_spreadsheet} */
```

Comment:
does it require a particular config file to work ?
It doesn't do anything on my side.

Reply:
it 'works" because I added types in the tsconfig file and inside files I have a folder of o_spreadsheet

Reply:
done :)